addons/web_editor/static/src/js/widgets/widgets.js

```diff
@@ -550,6 +573,93 @@ var ImageWidget = MediaWidget.extend({
         });
         this.$el.submit();
     },
+    /**
+     * @private
+     */
+    _loadMoreImages: function () {
+        this.IMAGES_ROWS = this.IMAGES_ROWS + 2;
+        this.IMAGES_PER_PAGE = this.IMAGES_ROWS * this.IMAGES_PER_ROW;
+        this._renderImages();
+        this._displayLoadMoreBtn();
+    },/**
+     * @private
+     */
+    _loadMoreUnsplash: function () {
+        this.IMAGES_UNSPLASH_ROWS = this.IMAGES_UNSPLASH_ROWS + 2;
+        this.IMAGES_UNSPLASH_PER_PAGE = this.IMAGES_UNSPLASH_ROWS * this.IMAGES_UNSPLASH_PER_ROW;
+        this._renderImages();
```

Comment:
Careful: no reference to unsplash in the web_editor module ! Everything which concerns unsplash should be in the web_unsplash module :wink: